client/blocks/reader-feed-header/index.jsx

```diff
@@ -55,7 +56,7 @@ class FeedHeader extends Component {
 		const ownerDisplayName =
 			site && ! site.is_multi_author && site.owner && site.owner.name;
 		const siteish = this.buildSiteish( site, feed );
-		const description = site && site.description;
+		const description = getSiteDescription( { site, feed } );
```

Comment:
weird, i feel like i fixed this in https://github.com/Automattic/wp-calypso/pull/14105/files ...

I guess it doesn't conflict because it's the same literal change? Maybe rebase this and see if it goes away?